src/dataDisplay/dataGrid/dataGridTableRow.jsx

```diff
@@ -140,7 +154,9 @@ class DataGridTableRow extends React.Component {
                                         />
                                     </Grid.Column>
 
-                                    <Grid.Column>
+                                    <Grid.Column
+                                        className={classes.accessorGridColumn}
+                                    >
                                         {accessor}
```

Comment:
"accessor" as name refers to the function or means of accessing something.
So, in my opinion, `accessor` is not a good identifier here for the data or value that has been accessed via the provided accessor (and thus `accessorGridColumn` is not a good class name identifier).

Could we rename these?  The name should indicate that this is the data or value to be displayed in the column/cell.
Maybe:
`accessor` => `accessedDataValue`
`classes.accessorGridColumn` => `classes.dataValueGridColumn`

Or something along those lines?
Possibly NAB, but this just stuck out at me, and I had to think a bit and look at other code.

Reply:
Sure...

Fixing.

Reply:
Fixed.

Reply:
I'll live with `accessorDataValue` :grin:
I would have preferred `accessedDataValue` (i.e. we used the accessor and have _accessed_ it, and here it is!)
But meh ... no biggie.
Resolving.